ee/clickhouse/sql/cohort.py

```diff
@@ -68,7 +68,7 @@
 """
 
 GET_PERSON_ID_BY_ENTITY_COUNT_SQL = """
-SELECT person_id FROM events
+SELECT pdi.person_id as person_id FROM events
```

Comment:
Seems something related to this change or something else where this query is used is failing. I've double checked that all changes are the same as #9251. @EDsCODE can you maybe take a look?